gui/workorder-details.lua

```diff
@@ -1,33 +1,18 @@
 -- adjust work orders' input item, material, traits
---[====[
 
-gui/workorder-details
-=====================
-Adjust input items, material, or traits for work orders. Actual
-jobs created for it will inherit the details.
-
-This is the equivalent of `gui/workshop-job` for work orders,
-with the additional possibility to set input items' traits.
-
-It has to be run from a work order's detail screen
-(:kbd:`j-m`, select work order, :kbd:`d`).
-
-For best experience add the following to your ``dfhack*.init``::
-
-    keybinding add D@workquota_details gui/workorder-details
-
-]====]
+--@ module = true
 
 --[[
 Credit goes to the author of `gui/workshop-job`, it showed
 me the way. Also, a huge chunk of code could be reused.
 ]]
 
+local overlay = require('plugins.overlay')
+
 local utils = require 'utils'
 local gui = require 'gui'
 local guimat = require 'gui.materials'
 local widgets = require 'gui.widgets'
-local dlg = require 'gui.dialogs'
 
 local wsj = reqscript 'gui/workshop-job'
 
```

Comment:
This would need to be updated to modern standards with a gui.ZScreen (maybe a gui.ZScreenModal for this) and widgets.Window. gui.FramedScreen is deprecated.